tests/cli/test_history.py

```diff
@@ -57,9 +57,15 @@ def test_build_history_query():
     # run buildtest history query <id>
     query_builds(build_id=build_id, log_option=False, output=False)
 
+    # run buildtest history --pager query <id>
```

Comment:
just get rid of the `run` from all the comments so we can see the actual command